cognite/client/_api/entity_matching.py

```diff
@@ -141,7 +141,7 @@ def fit(
         capabilities in the project, are able to access the data sent to this endpoint.
 
         Args:
-            sources: entities to match from, should have an 'id' field. Tolerant to passing more than is needed or used (e.g. json dump of time series list)
+            sources: entities to match from, should have an 'id' field. Tolerant to passing more than is needed or used (e.g. json dump of time series list). Metadata fields are flattened to "metadata.key" entries for use in match_fields.
```

Comment:
This kind of makes it sound like the user is supposed to do that, but the code looks like it does it for the user.

Reply:
rephrased